plugins/auth-node/api-report.md

```diff
@@ -4,6 +4,7 @@
 
 ```ts
 import { PluginEndpointDiscovery } from '@backstage/backend-common';
+import { Request as Request_2 } from 'express';
```

Comment:
Wondering if we have to use the `express` types here? And not just the basic `Request` type?

Reply:
Done.

Reply:
The difference is minimal `request.headers.authorization` vs `request.headers.get('authorization')` so I switch to the simpler interface.

Reply:
I wasn't able to get this to work with the node-fetch Request because express is passing down the express.Request type. Id love to hear your thoughts on this..